extensions/interactions/customization-args-defs.ts

```diff
@@ -201,78 +201,78 @@ export interface GraphInputCustomizationArgs {
 
 
 interface ImageClickInputCustomizationArgsBackendDict {
```

Comment:
@nishantwrp Hi Nishant, I have a question here.. why is highlightRegionsOnHover here a string for ImageClickInputCustomizationArgsBackendDict, but a boolean for ImageClickInputCustomizationArgs? Ditto for InteractiveMapCustomizationArgsBackendDict (zoom is a string on backend dict, but a number for frontend)

Reply:
For now, I've changed the backend dicts to match the frontend ones. Let me know if that's a bad change.

Reply:
yup, it should be boolean in noth

Reply:
Gotchu- thanks for the quick response